CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+# Meraki Front-end Repository
+
+This repository contains the source code for the front-end of the Meraki web application, which has enabled over **25,000 students** from underserved communities to learn the basics of programming and spoken English through free online classes and curricula materials. Your assistance will allow us to continue to scale quickly and help reach our vision of building a sustainable skill based learning platform that can enable **1M children** from the **most underserved** communities of our society find **aspirational jobs** in tech by the year 2025.
+
+## Meraki App Links
+
+Production (Deployed from the [`main`](https://github.com/navgurukul/bhanwari-devi) branch): [https://merakilearn.org](https://merakilearn.org)
+
+Development (Deployed from the [`dev`](https://github.com/navgurukul/bhanwari-devi/tree/dev) branch): [https://bhanwari-devi.vercel.app](https://bhanwari-devi.vercel.app)
+
+## Tech Stack
+
+- [React](https://reactjs.org/)
+- [Redux](https://redux.js.org/)
+- [MUI](https://mui.com/)
+- [Node.js](https://nodejs.org/)
+
+## Installation and Setup Guide
+
+1. Clone and Fork the `navgurukul/bhanwari-devi` repository. If you [download GitHub Desktop](https://desktop.github.com), you can refer to this [link](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop) for directions on how to do this.
+2. If you're using GitHub Desktop, switch to the `dev` branch. If you're using [Git Bash](https://git-scm.com/downloads), you can enter the following:
+```
+# first time only to alias the NG repository
+git remote add upstream https://github.com/navgurukul/bhanwari-devi.git
+
+# switches to that branch
+git checkout dev
```

Comment:
May want to add the fact that they should create their own feature branch based on the dev branch and create PR.